server/services/transport/daily.mdx

```diff
@@ -916,6 +916,97 @@ See [update_subscriptions](https://reference-python.daily.co/api_reference.html#
 
 </ResponseField>
 
+### Permissions Management
+
+When using the DailyTransport, if you provide a token with owner permissions to the bot, it can control permissions for other participants in the session. With this control, the bot can modify settings for how others send and receive media, how they are seen in a session, and whether they have Administrative controls.
+
+<ResponseField name="update_remote_participants" type="async method">
+  Updates permissions and settings for specific participants in the room.
+
+```python
+async def update_remote_participants(remote_participants: Mapping[str, Any] = None) -> None
+```
+
+**Parameters:**
+
+- `remote_participants`: Dictionary mapping participant IDs to their updated settings
+
+This method allows fine-grained control over participant capabilities and permissions in the Daily room.
+
+**Permission Structure:**
+
+The permission system follows a hierarchical structure:
+
+- `RemoteParticipantUpdates`: Maps participant IDs to their updates
+  - `ParticipantUpdate`: Contains permissions and input settings
+    - `permissions`: Defines what a participant can do
+      - `hasPresence`: Whether participant appears in the room
+      - `canAdmin`: Administrative permissions (participants, streaming, transcription)
+      - `canSend`: Media types the participant can send ("camera", "microphone", etc.)
+      - `canReceive`: Controls whose media the participant can receive
+        - `base`: Default receive permission for all participants
+        - `byUserId`: User-specific receive permissions
+        - `byParticipantId`: Participant-specific receive permissions
+    - `inputsEnabled`: Controls whether inputs are enabled for the participant
+
+**Examples:**
+
+```python
+# Mute a participant's microphone
+await transport.update_remote_participants({
+    "participant-123": {
+        "inputsEnabled": {
+            "microphone": False
+        }
+    }
+})
+
+# Revoke sending permissions
+await transport.update_remote_participants({
+    "participant-123": {
+        "permissions": {
+            "canSend": []  # Empty list revokes all send permissions
+        }
+    }
+})
+
+# Configure selective audio routing with custom streams
+await transport.update_remote_participants({
+    "customer-id": {
+        "permissions": {
+            "canReceive": {
+                "base": False,  # Don't receive from anyone by default
+                "byUserId": {
+                    "bot": {
+                        "customAudio": {"hold-music": True}  # Only receive hold music
```

Comment:
I know this came from the warm transfer demo, but I had to actually change that demo when I realized we don't actually support custom tracks in daily-python. The workaround was to spawn a separate daily-python participant that just sent hold music. So maybe this example could be reworked to:

```json
{
  "base": False, # Don't receive any media from anyone by default
  "byUserId": {
    "hold-music": True
  }
}
```

Reply:
Updated. Commit incoming.